_posts/2020-08-03-roadmap.md

```diff
@@ -0,0 +1,23 @@
+
+In 1.0.0 we also began to explore an exciting new approach: direct interoperability with other popular packages for molecular analysis. [Converters](https://www.mdanalysis.org/docs//documentation_pages/converters.html) are distinct from [topology parsers](https://www.mdanalysis.org/docs//documentation_pages/topology_modules.html) and [coordinate readers](https://www.mdanalysis.org/docs//documentation_pages/coordinates_modules.html) as a third avenue for loading data into MDAnalysis. In 1.0.0 we added converters for two libraries: the molecular editor ParmEd, and chemfiles, a library for reading data from computational chemistry formats.
+
+The general lack of interoperability between software packages in the molecular modelling community has been highlighted in the [2019 report of the NSF MolSSI on Molecular Dynamics Software Interoperability](https://drive.google.com/file/d/1HKRhfm1Ev1UMAdvvuXhorSCUJlhtU-1o/view), noting consequences such as great duplication of effort in developing and maintaining similar tools across different formats; significant barriers to collaborating and transferring data; and requiring scientists to learn multiple packages and languages to access the full breadth of available analysis algorithms. 
+
+Moving forward, our plan is to increase the range of analyses and formats accessible to users by becoming interoperable with other relevant libraries. This reduces the need to duplicate and support tools within our own framework, and allows MDAnalysis to become a general-purpose analysis toolkit. We are already in the process of expanding compatible libraries in 2.0.0 by adding support for the widely popular [RDKit](https://www.rdkit.org/) cheminformatics toolkit via Cedric's [Google Summer of Code project](https://www.mdanalysis.org/2020/05/20/gsoc-students/#c%C3%A9dric-bouysset-from-rdkit-to-the-universe-and-back).
+
+Other targets for future releases include widely-used analysis libraries, such as [MDTraj](http://mdtraj.org/); libraries that can expand the range of formats we can support, such as [OpenBabel](http://openbabel.org/wiki/Main_Page); and direct interfacing with MD engines such as [OpenMM](http://openmm.org/).
+
+If you have comments or suggestions for where else we can look, then please get in touch!
```

Comment:
I feel like this could be worded somehow a little bit better. I was thinking something along the lines of "Ensuring robust interoperability is by design a community effort If you are interested in contributing or just have comments/suggestions on our future directions, please get in touch!"

Reply:
I like your "community can-do" attitude, @IAlibay